geode-management/src/main/java/org/apache/geode/management/configuration/Region.java

```diff
@@ -213,4 +224,74 @@ public void setAction(ExpirationAction action) {
       this.action = action;
     }
   }
+
+  public enum EvictionType {
+    ENTRY_COUNT,
+    MEMORY_SIZE,
+    HEAP_PERCENTAGE
+  }
+
+  public enum EvictionAction {
+    LOCAL_DESTROY,
+    OVERFLOW_TO_DISK
+  }
+
+  public static class Eviction implements Serializable {
+    private EvictionType type;
+    private EvictionAction action;
+    private Integer entryCount;
+    private Integer memorySizeMb;
+    private String objectSizer;
+
+    public Eviction() {}
+
+    public Eviction(EvictionType type, EvictionAction action, Integer entryCount,
+        Integer memorySizeMb, String objectSizer) {
+      this.type = type;
+      this.action = action;
+      this.entryCount = entryCount;
+      this.memorySizeMb = memorySizeMb;
+      this.objectSizer = objectSizer;
+    }
+
+    public EvictionType getType() {
+      return type;
+    }
+
+    public void setType(EvictionType type) {
+      this.type = type;
+    }
+
+    public EvictionAction getAction() {
+      return action;
+    }
+
+    public void setAction(EvictionAction action) {
+      this.action = action;
+    }
+
+    public Integer getEntryCount() {
+      return entryCount;
+    }
+
+    public void setEntryCount(Integer entryCount) {
+      this.entryCount = entryCount;
```

Comment:
I thought setEntryCount would also set the type implicitly.

Reply:
It does get set in the cluster configuration after conversion.